resources/js/Components/NavBar/NavBar.vue

```diff
@@ -31,14 +32,17 @@
 </script>
 
 <template>
-    <nav class="navbar bg-base-100 lg:px-4">
+    <nav class="navbar lg:px-4">
         <div class="navbar-start">
-            <a class="btn btn-ghost normal-case text-lg hidden lg:block" href="/" ><img class="h-8 w-auto" :src="'images/logo-blue.svg'" /></a>
+            <a class="btn btn-ghost normal-case text-lg hidden lg:block" href="/" >
+                <img v-show="this.dark" class="h-8 w-auto" :src="'images/logo-blue-white.svg'" />
+                <img v-show="!this.dark" class="h-8 w-auto" :src="'images/logo-blue.svg'" />
```

Comment:
Nesta situação podemos passar diretamente a prop para a tag `img`. Por norma os [bind props são passadas quando não estás a passar strings](https://vuejs.org/guide/components/props.html#prop-passing-details) (números, funções, etc).

```suggestion
                <img v-show="this.dark" class="h-8 w-auto" src="images/logo-blue-white.svg" />
                <img v-show="!this.dark" class="h-8 w-auto" src="images/logo-blue.svg" />
```